gettsim/parameters/eink_st_abzuege.yaml

```diff
@@ -381,6 +381,37 @@ altersentlastung_quote:
     scalar: 0.136
     reference: Artikel 1 G. v. 05.07.2004 BGBl. I S. 1427.
 
+altersentlastungsbetrag_altersgrenze:
+  name:
+    de: Altersgrenze für Altersentlastungsbetrag
+    en: Age limit for old-age relief amount
+  description:
+    de: Der Altersentlastungsbetrag wird einem Steuerpflichtigen gewährt,
+        der vor dem Beginn des Kalenderjahres, in dem er sein Einkommen bezogen hat,
+        das 64. Lebensjahr vollendet hatte. Zuvor wurde er als Altersfreibetrag bezeichnet.
```

Comment:
"Zuvor" = until when?

Reply:
I couldn't find any information on this. It seems that "Altersfreibetrag" is more of a colloquial term for the official term "Altersentlastungsbetrag". Shall we just delete this sentence or replace it by something like "Es handelt sich hierbei um einen Altersfreibetrag."?

Reply:
Just delete! Maybe just a reference to former name in the code? Does not matter.